### DISCUSSION TITLE
Add inotify support?

### DISCUSSION THREAD

**Discussion by k4my4b** [nextcloud/all-in-one#1288] [Ideas] [answered]
Is there a plan to include support for `inotify` and by extension `INotify file watcher`
The intotify app is currently the only reasonable approach to dealing with local external storage mediums and/or shared folders with other services e.g. a samba share.

**Comment by YeapGuy** [comment, marked as answer]
This should be possible with #1163 (respective issue: #1162) once it's merged

**Reply by szaimen** [reply]
Yes, indeeed

**Reply by szaimen** [reply]
This is now released with v3.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by k4my4b** [reply]
> This is now released with v3.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

Hi I can't seem to find any new/additional option(s) to add dependencies :confused: 
(I can see $STARTUP_APPS has been added but is there no addition to the AIO UI itself?)

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one#how-to-add-php-extensions-permanently-to-the-nextcloud-container

**Reply by Aquariu** [reply]
Hey ! Sorry to dig up an old issue.
What is the best way to run the files_external:notify occ commands regularly ? I guess I could run a crontab line on the host with a docker exec command. However, with the daily borg backup I'm nevery exactly sure as to when the Nextcloud instance is running again and when Ishould launch those commands. Is there a way to have those commands launched automatically at the start of the instance ?

**Comment by johnnyq** [comment]
Would this also relate to external storage FTP as well?

**Reply by szaimen** [reply]
> Would this also relate to external storage FTP as well?

No, external FTP storage does not support inotify. It is only available for external SMB storage and local external storage using the files_intotify app. But you could probably mount the FTP connection via the OS and use local external storage and the files_inotify app with that. But I never tried that...

**Reply by johnnyq** [reply]
Thanks @szaimen =]
